keyboards/miuni32/keyboard.json

```diff
@@ -3,8 +3,8 @@
     "manufacturer": "Bigtuna.io",
     "maintainer": "qmk",
     "usb": {
-        "vid": "0xFEED",
-        "pid": "0x6060",
+        "vid": "0x21FF",
```

Comment:
This is an already allocated VendorID, which should be avoided. The general convention is to use something over `0x4000`.

Reply:
Good catch! Thanks, will change it